JSON/Mystery Widget.json

```diff
@@ -0,0 +1,23 @@
+{
+  "Author": "BakersDozenBagels",
+  "Compatibility": "Compatible",
+  "DefuserDifficulty": "Easy",
+  "Description": "Hides a widget until certain modules are solved. Tags: Push, Hidden-Widget",
+  "ExpertDifficulty": "VeryEasy",
+  "Ignore": [ "The Heart", "Turn The Keys", "+FullBoss", "+SemiBoss"],
```

Comment:
Needs separate lines instead of one line. Putting it through an auto-formatter should do it.